lib/jekyll/post.rb

```diff
@@ -118,6 +119,14 @@ def title
       data.fetch('title') { titleized_slug }
     end
 
+    # Public: the Post excerpt_separator, from the YAML Front-Matter or site default
+    #         excerpt_separator value
+    #
+    # Returns the post excerpt_separator
+    def excerpt_separator
+      data.fetch('excerpt_separator') { site.config['excerpt_separator'].to_s }
```

Comment:
I think it would seem a bit simpler to write `data.fetch('excerpt_separator', site.config['excerpt_separator'].to_s)` instead of using the block.